pmkoalas/models/petrinet.py

```diff
@@ -0,0 +1,341 @@
+
+# 
+# Weighted Petri net with place and transition labels.
+# 
+# Allows explicit place or transition ids for simpler comparison, especially
+# during testing.
+#
+
+import operator
```

Comment:
Why is this library imported? I cannot find where it has been used

Reply:
Looks like it is unused.